src/css.js

```diff
@@ -10,17 +10,16 @@ define([
 	"./css/var/getStyles",
 	"./css/curCSS",
 	"./css/adjustCSS",
-	"./css/defaultDisplay",
 	"./css/addGetHookIf",
 	"./css/support",
-	"./data/var/dataPriv",
+	"./css/showHide",
```

Comment:
So, introduce a new showHide module (`.show`, `.hide`, and `.toggle`) depending on data and depended upon by effects? That sounds reasonable, but out of scope for this ticket.